modules/dists/StencilDist.chpl

```diff
@@ -1562,7 +1561,7 @@ proc StencilArr.dsiNoFluffView() {
     }
   }
   if doRADOpt then alias.setupRADOpt();
-  newDom.add_arr(alias);
+  newDom.add_arr(alias, locking=false);
```

Comment:
I like that you commented why the lock isn't needed in the other places you added the "locking=false".  I think it'd be good to add that comment here, too